### PR TITLE
DOCS-Delete-etcd3-reference

### PR DESCRIPTION
**Pull request by bmckercher123**
Request from PM on behalf of customer to remove reference to etcd3 in CE doc sys reqs. (Created separate doc tickets for full review of CE and OSS sys reqs per Casey's finding other things to fix.) 

Product Version(s):
All CE unarchived versions

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.17/_includes/components/ReqsSys.js [hunk 1]
```diff
@@ -181,8 +181,7 @@ function KeyValueStore(props) {
             OpenShift: <span>With OpenShift, the Kubernetes API datastore is used for the key/value store.</span>,
             Kubernetes: (
               <span>
-                On Kubernetes, you can configure {prodname} to access an etcdv3 cluster directly or to use the
-                Kubernetes API datastore.
+                On Kubernetes, you can configure {prodname} to access a cluster directly or to use the Kubernetes API datastore.
```
Comment (caseydavenport) [inline calico-enterprise_versioned_docs/version-3.17/_includes/components/ReqsSys.js]: I'm not sure this is the right fix. I think we can probably just remove the data store requirement altogether from enterprise? There's only ever one option (kdd).

Similarly, "access a cluster directly or to use the Kubernetes API datastore." is somewhat redundant - the Kubernetes API data store is accessing the API server(s) directly.

This section also talks about OpenStack, which isn't supported in Enterprise either.

Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.17/_includes/components/ReqsSys.js]: Deleted the function entirely for CE.

File: calico-enterprise/_includes/components/ReqsSys.js [hunk 2]
```diff
@@ -165,41 +165,6 @@ function NodeRequirementsEnt(props) {
   );
 }
 
-function KeyValueStore(props) {
```
Comment (caseydavenport) [inline calico-enterprise/_includes/components/ReqsSys.js:168]: I'm not really sure how this works - is this function called from somewhere that we need to remove it as well?

**Review by caseydavenport — APPROVED** [review]
Technically looks good :+1:

File: calico-enterprise/_includes/components/ReqsSys.js (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise/_includes/components/ReqsSys.js:168]: Yeah, it is failing to build. Need to track down where its called.

**Comment by ctauchen** [comment]
@bmckercher123 LGTM. Looks like this succeeds in removing §Datastore requirements altogether.

**Pull request merged**